lib/hash-feed.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const { IdHasher } = require('asset-pipe-common');
+const { Readable } = require('readable-stream');
+
+/**
+ * Hashes a feeds ids together
+ *
+ * @param {object[]} feed - browserify compatible bundle feed
+ * @returns {Promise<string>} - hash of all id properties in given feed
+ */
+module.exports = function hashFeed(feed) {
+    const source = new Readable({ objectMode: true, read() {} });
+    const hasher = new IdHasher();
+
+    source.pipe(hasher);
+
+    return new Promise(resolve => {
+        hasher.on('finish', () => {
```

Comment:
can't we just `await getStream(hasher)`?

Reply:
awww yea, will try

Reply:
no, not really. Or at least not in a way that cleans much up. Still need to push items into a stream and get the .hash property at the end.

Reply:
yeah, but using `getStream` would allow us to drop error handling, as it's done for us. What if the readstream fails?